hyppo/independence/hhg.py

```diff
@@ -4,7 +4,9 @@
 
 from ..tools import compute_dist
 from ._utils import _CheckInputs
-from .base import IndependenceTest
+from .base import IndependenceTest, IndependenceTestOutput
+from scipy.spatial.distance import cdist
```

Comment:
why? we already have a pairwise distance matrix function

Reply:
Function removed.